docs/api/web-contents.md

```diff
@@ -151,6 +151,9 @@ Returns:
   [`BrowserWindow`](browser-window.md).
 * `additionalFeatures` String[] - The non-standard features (features not handled
   by Chromium or Electron) given to `window.open()`.
+* `referrer` [Referrer](structures/referrer.md) - The referrer that will be
```

Comment:
Not sure if this is relevant, but I remember that there was a historical misspelling of the `referer` HTTP header. Are we affected by this? Should we also be spelling it "wrong"?

> The misspelling of referrer originated in the original proposal by computer scientist Phillip Hallam-Baker to incorporate the field into the HTTP specification.[3] The misspelling was set in stone by the time of its incorporation into the Request for Comments standards document RFC 1945

Reply:
Almost all other documentation that refers to the `Referer` header uses `Referrer` wherever it can. The HTTP header for referrer policy is called `Referrer-Policy`, without the misspelling. The web community as a whole appears to deeply regret the error, and prefers to pretend it never happened :D